src/index.html

```diff
@@ -8,6 +8,87 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="content">
+      <!-- star 1 --->
+      <div class="stars stars--0">
+        <div class="stars__star">
+        </div>
+        <div class="stars__star">
+        </div>
+        <div class="stars__star">
+        </div>
+        <div class="stars__star">
+        </div>
+        <div class="stars__star stars__star--last">
```

Comment:
why do we need this `stars__star--last` class name?

Reply:
removed, thanks